internal/repository/prisma/dbsqlc/workflow_runs.sql

```diff
@@ -242,13 +242,16 @@ WITH jobRuns AS (
         sum(case when runs."status" = 'FAILED' then 1 else 0 end) AS failedRuns,
         sum(case when runs."status" = 'CANCELLED' then 1 else 0 end) AS cancelledRuns
     FROM "JobRun" as runs
+    JOIN "Job" as job ON runs."jobId" = job."id"
     WHERE
         "workflowRunId" = (
             SELECT "workflowRunId"
             FROM "JobRun"
             WHERE "id" = @jobRunId::uuid
         ) AND
-        "tenantId" = @tenantId::uuid
+        runs."tenantId" = @tenantId::uuid AND
+        -- we should not include onFailure jobs in the calculation
+        job."kind" != 'ON_FAILURE'
```

Comment:
Should we make this only including defaults i would think future cases will also need to filtered?

Reply:
Good catch, will fix